src/web/app/components/logs-table/logs-table.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { LogsTableRowModel } from './logs-table-model';
+
+/**
+ * A table to display logs.
+ */
+@Component({
+  selector: 'tm-logs-table',
+  templateUrl: './logs-table.component.html',
+  styleUrls: ['./logs-table.component.scss'],
+})
+export class LogsTableComponent implements OnInit {
+
+  @Input()
+  logs: LogsTableRowModel[] = [];
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  expandDetails(logsTableRowModel: LogsTableRowModel): void {
+    logsTableRowModel.isDetailsExpanded = !logsTableRowModel.isDetailsExpanded;
+  }
+
+  getClassForStatus(httpStatus: number): string {
+    const num: number = Math.floor(httpStatus / 100);
+    if (num === 2) {
+      return 'green-font';
+    }
+    if (num === 4) {
+      return 'orange-font';
+    }
+    if (num === 5) {
+      return 'red-font';
+    }
+    return '';
+  }
+
+  getClassForSeverity(severity: string): string {
+    if (severity === 'INFO') {
```

Comment:
Similarly, suggest to use switch case. There are log levels lower than `INFO` and higher than `ERROR` that we may or may not be using.